polaris/ocean/tests/baroclinic_channel/forward.yaml

```diff
@@ -5,7 +5,8 @@ omega:
     config_use_mom_del2: true
     config_mom_del2: 10.0
   bottom_drag:
-    config_implicit_bottom_drag_coeff: 0.01
+    config_implicit_bottom_drag_type: constant
```

Comment:
Constant drag is the default so technically this line isn't needed but we can include it anyhow

Reply:
I'm happy to leave it or take it out.  I was following what was done in the `inertial_gravity_wave` test group.